.github/workflows/integration-tests-reusable.yml

```diff
@@ -131,18 +131,19 @@ jobs:
         DAFNY_RELEASE: ${{ github.workspace }}\unzippedRelease\dafny
       run: |
         cmd /c mklink D:\a\dafny\dafny\unzippedRelease\dafny\z3\bin\z3-4.8.5 D:\a\dafny\dafny\unzippedRelease\dafny\z3\bin\z3-4.8.5.exe
-        dotnet test --logger trx --logger "console;verbosity=normal" dafny/Source/IntegrationTests
+        dotnet test --logger trx --logger "console;verbosity=normal" --collect:"XPlat Code Coverage" dafny/Source/IntegrationTests/IntegrationTests.csproj
```

Comment:
Does this actually measure coverage of the integration tests, given that most of the code is run in separate processes and not the main testing process?

Reply:
I was impressed to find that, yes, it does. We get over 80% coverage of DafnyCore just from this one `dotnet test` command.